zScanner/Views/NewDocument/PhotoSelectorCollectionViewCell.swift

```diff
@@ -75,6 +94,15 @@ class PhotoSelectorCollectionViewCell: UICollectionViewCell {
         return image
     }()
     
+    private lazy var textField : UITextField = {
+        let text = UITextField()
+        text.placeholder = "newDocumentPhotos.description.placeholder".localized
```

Comment:
setup font explicitly to respect "Dynamic Type". Use one of the fonts specified in `UIFont.swift` extension